tests/testDbLocal.py

```diff
@@ -471,12 +470,10 @@ def testLoadDataInFile(self):
 
 ####################################################################################
 def main():
-    logging.basicConfig(
-        format='%(asctime)s %(name)s %(levelname)s: %(message)s',
-        datefmt='%m/%d/%Y %I:%M:%S',
-        level=logging.DEBUG)
+    log.configure()
 
-    credFile = os.path.expanduser(TestDbLocal.CREDFILE)
+    if TestDbLocal.CREDFILE.startswith('~'):
```

Comment:
This is something that we talked about last week - if TestDbLocal.CREDFILE does not start with `~` then `credFile` is undefined which will result in exception. I think it's always OK to do `expanduser()` even if it does not have `~`.